docs/source/contributing/dev-contributions/index.rst

```diff
@@ -14,8 +14,8 @@ Developer Guide
    releasing.rst
```

Comment:
The Developer Guide is well written and provides clear instructions. However, there are a few minor changes I would suggest:

1. In the first paragraph, consider replacing "Thanks for your contribution!" with "We appreciate your contribution!" for a more formal tone.

2. In the second paragraph, the phrase "each project has documentation on ReadTheDocs" might be better as "documentation for each project can be found on ReadTheDocs".

3. For the code block, consider adding a brief explanation about what `pip install .` does for those who might not be familiar with it.

4. In the last paragraph, the phrase "the information about that is in the `notebook contributor documentation`" could be rephrased for clarity. Consider "you can find more information about this in the `notebook contributor documentation`". 

5. Lastly, consider adding a conclusion or closing statement to wrap up the guide.